internal/cluster/validator.go

```diff
@@ -566,3 +573,36 @@ func (v *clusterValidator) isNtpServerConfigured(c *clusterPreprocessContext) (V
 	return ValidationFailure, fmt.Sprintf("Hosts' clocks are not synchronized (there's more than a %d minutes gap between clocks), "+
 		"please configure an NTP server via DHCP or set clocks manually.", common.MaximumAllowedTimeDiffMinutes)
 }
+
+// Ignore hostPrefix set to 0 for non-OVN/SDN plugins.
+func (v *clusterValidator) skipNetworkHostPrefixCheck(c *clusterPreprocessContext, prefix int64) bool {
+	// list of known plugins that require hostPrefix to be set
+	var pluginsUsingHostPrefix = []string{models.ClusterNetworkTypeOVNKubernetes, models.ClusterNetworkTypeOpenShiftSDN}
+
+	if prefix != 0 {
+		return false
+	}
+
+	if !funk.ContainsString(pluginsUsingHostPrefix, swag.StringValue(c.cluster.NetworkType)) {
```

Comment:
currently this condition will we always false as we allow only 2 network types, so maybe there is no point to validate it?

Reply:
I was mainly added this in response to the comment about adding Calico or a customer network to the network list https://issues.redhat.com/browse/OCPBUGS-19823?focusedId=23278074&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23278074
If that was done in the future this check would not have to change.